WebRoot/h/changepass

```diff
@@ -179,6 +180,14 @@
                                             ${minDigitsOrPuncsMsg}
                                         </li>
                                     </c:if>
+                                    <c:if test="${zm:boolean(zimbraPasswordAllowUsername)}">
```

Comment:
Is the condition check correct?  If it is true, username is allowed and then the error message should not appear...?

Reply:
This condition was changed to check the correct value